deploy/clusterlink-agent.yaml

```diff
@@ -28,6 +28,7 @@ spec:
           command:
             - clusterlink-agent
             - -kubeconfig=/etc/clusterlink/kubeconfig
+            - --v=4
```

Comment:
what does `v` mean ？

Reply:
> what does `v` mean ？klog level ？ why do you set this value but not modify the level which you want print ？

```v```  represents the log level.  The code is configured at this level, but I forgot to configure it in the yaml file

Reply:
What I mean is why we set `4`， It may output many debug logs